catalog/resource_sql_table.go

```diff
@@ -406,6 +430,23 @@ func ResourceSqlTable() common.Resource {
 	return common.Resource{
 		Schema: tableSchema,
 		CustomizeDiff: func(ctx context.Context, d *schema.ResourceDiff) error {
+			if d.HasChange("column") {
+				old, new := d.GetChange("column")
+				oldCols := old.([]interface{})
+				newCols := new.([]interface{})
```

Comment:
I think these are `[]map[string]any`, let's use that instead, saving you the second typecheck below.
```suggestion
				oldCols := old.([]map[string]any)
				newCols := new.([]map[string]any)
```

Reply:
Tried this but ran into some errors doing type casting, so going back to my old approach